src/GeneralStateTestsFiller/stTransactionTest/PointAtInfinityECRecoverFiller.yml

```diff
@@ -0,0 +1,59 @@
+# Call ECRecover on invalid msg, v, r, s
+---
+PointAtInfinityECRecover:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    000000000000000000000000000000000000ff0a:
+      balance: '0xffffffff'
+      code: ':yul {
+        mstore(0, 0x6b8d2c81b11b2d699528dde488dbdf2f94293d0d33c32e347f255fa4a6c1f0a9)
+        mstore(32, 0x1b)
+        mstore(64, 0x79be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798)
+        mstore(96, 0x6b8d2c81b11b2d699528dde488dbdf2f94293d0d33c32e347f255fa4a6c1f0a9)
+        sstore(0, call(1000000, 1, 0, 0, 128, 0, 32))
+        sstore(1, mload(0))
+      }'
+      nonce: 0
+      storage: {}
+
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '0xde0b6b3a7640000'
+      code: ''
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      - ''
+    gasLimit:
+      - 10000000
+    gasPrice: 10
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
```

Comment:
here is also must be supported the v,r,s format. 
try adding up invalid signature transaction test, see if it works with expect exception.